extensions/python/ferrox_notebooks/06_phase_classification.py

```
@@ -0,0 +1,609 @@
+    cutoff_slider = mo.ui.slider(2.5, 6.0, value=4.0, step=0.25, label="Cutoff (Å)")
+    q_degree_slider = mo.ui.slider(4, 12, value=6, step=2, label="Q degree")
+
+    return cutoff_slider, q_degree_slider
+
+
+@app.cell
+def _(bcc, cutoff_slider, fcc, ferrox, hcp, mo, order_params, q_degree_slider, time):
+    """Calculate Steinhardt Q parameters with interactive controls."""
+
+    cutoff = cutoff_slider.value
+    q_degree = q_degree_slider.value
```

Ruff (0.14.14)

[error] Local variable `q_degree` is assigned to but never used

Remove assignment to unused variable `q_degree`

(F841)